cmd/res/configuration.toml

```diff
@@ -1,5 +1,5 @@
 [Writable]
-LogLevel = 'INFO'
+LogLevel = 'DEBUG'
```

Comment:
change back to INFO

Reply:
Yes I thought I did that I will change it.